client_lib/pulp/client/commands/repo/sync_publish.py

```diff
@@ -190,13 +193,14 @@ def __init__(self, context, renderer, distributor_id, name='run', description=DE
                                         options will override respective options from the default
                                         publish config. Each entry should be either a PulpCliOption
                                         or PulpCliFlag instance
-        :type override_config_options: list
+        :type override_config_options: tuple
         """
         super(RunPublishRepositoryCommand, self).__init__(name, description, method, context,
                                                           renderer)
 
         self.distributor_id = distributor_id
         self.override_config_keywords = []
+        override_config_options += (FLAG_FORCE_FULL, )
```

Comment:
@mhrivnak ,so are we fine to merge these 3 related PRs once i will add docs as mentioned @bmbouter here?https://github.com/pulp/pulp/pull/2548#issuecomment-212457808